include/deal.II/distributed/tria.h

```diff
@@ -880,7 +881,37 @@ namespace parallel
         std::map<unsigned int, pack_callback_t> pack_callbacks;
       };
 
-      CellAttachedData cell_attached_data;
+      /**
+       * Vector of tuples containing of p4est quadrant, dealii cells and their relation after refinement.
+       * This makes a one to one correlation possible and allows for parallelization in the future.
```

Comment:
drop the comment about the future -- the future always becomes outdated so quickly...